### ISSUE TITLE
Sublime Text 3 does not show plots on Windows

### ISSUE THREAD

**Issue by tomvanmele** [compas-dev/compas#263]
**Description**
On windows, Sublime Text 3 now apparently suppresses GUIs. Therefore, COMPAS plotter windows will not appear and the execution of the script never terminates (you have to kill the Python process using your task manager).  

**To Reproduce**
1. In Sublime Text 3
2. sample.py (see below)

**Expected behavior**
The plotter window should pop up and display the remeshing process.

**Desktop (please complete the following information):**
 - OS: Windows
 - Python version: any
 - Python package manager: any

**Comment by tomvanmele** [comment]
```python
# sample.py

from compas.datastructures import Mesh
from compas.datastructures import mesh_smooth_area
from compas.datastructures import trimesh_remesh
from compas.plotters import MeshPlotter

vertices = [(0.0, 0.0, 0.0), (10.0, 0.0, 0.0), (6.0, 10.0, 0.0), (0.0, 10.0, 0.0)]
faces = [[0, 1, 2, 3]]

mesh = Mesh.from_vertices_and_faces(vertices, faces)

key = mesh.insert_vertex(0)
fixed = [key]

plotter = MeshPlotter(mesh, figsize=(10, 7))

plotter.draw_edges(width=0.5)

def callback(mesh, k, args):
    print(k)
    plotter.update_edges()
    plotter.update()

trimesh_remesh(
    mesh,
    0.5,
    kmax=200,
    allow_boundary_split=True,
    allow_boundary_swap=True,
    allow_boundary_collapse=True,
    fixed=fixed,
    callback=callback)

mesh_smooth_area(mesh, fixed=mesh.vertices_on_boundary())

mesh_smooth_area(mesh, fixed=mesh.vertices_on_boundary())

plotter.update_edges()
plotter.update(pause=2.0)
plotter.show()
```

**Comment by tomvanmele** [comment]
the problem can be solved by adding a custom builder. unfortunately, that means you can't use *Conda* as your build system...

```
{
	"cmd":
	[
		"start",
		"cmd",
		"/k",
		"%USERPROFILE%\\Anaconda3\\python",
		"-u",
		"$file"
	],
	"shell": true,
	"file_regex": "^[ ]*File \"(...*?)\", line ([0-9]*)",
	"name": "ForGUIs",
	"selector": "source.python"
}

```

**Comment by gonzalocasas** [comment]
PR submitted to the `sublime-text-conda` package ☝️

**Comment by tomvanmele** [comment]
oooh nice!

**Comment by gonzalocasas** [comment]
Merged and released on [sublime-text-conda v0.5.2](https://github.com/mandeep/sublime-text-conda/releases/tag/v0.5.2).
